main-command/src/main/scala/sbt/BasicCommands.scala

```diff
@@ -39,7 +39,6 @@ object BasicCommands {
     ignore,
     help,
     completionsCommand,
-    multi,
```

Comment:
What's the reason for removing here?

Reply:
Yeah. I meant to put this in the commit message. If multi comes after the action parser in the parser list, and the command does not have a leading `;`, then the action parser will parse it before the multi parser has a chance. I'm not sure if there is a better way to handle this.

Reply:
In other words, we are introducing an ambiguity in the parsing semantics that's resolved by the implementation detail of which parser is listed first.

Reply:
Correct. I considered modifying the action parser, but that was a lot more work/was more scary.

Reply:
The reason I think this is ok is that the multi parser will not accept any command that doesn't contain a `;` that is not in a string literal.

Reply:
@eatkins
 
> The reason I think this is ok is that the multi parser will not accept any command that doesn't contain a `;` that is not in a string literal.

That's good.

@dwijnand 

> I think that's overly pessimistic sounding.
>
> This change is enhancing the action parser by allowing multiple actions to be defined.

Yea. I am all for treating `;` as part of command syntax, but I just want to understand if we're introducing some tradeoff. If Ethan's saying that `multi` won't accept unless there's `;`, that makes me feel better since that's not ambiguous when you have command like `testOnly foo`.

Reply:
Yeah. If that case were ambiguous, I think that sbt would get caught in an infinite loop.